databuilder/setup.py

```diff
@@ -74,13 +74,17 @@
     'apache-atlas>=0.0.11'
 ]
 
+oracle = [
+    'cx_Oracle==8.2.1'
```

Comment:
Does this require any specific Oracle drivers to be on the path to run or are all of the drivers included in the distribution of the `cx_oracle` package?

Reply:
Yes. The cx_Oracle module loads Oracle Client libraries which communicate over Oracle Net to an existing database.